### PR TITLE
Moved logout button to sidebar and fix default width sidebar.

### PR DESCRIPTION
**Pull request by dima-bzz**

### CODE REVIEW

**Comment by amir20** [comment]
A lot going on here so give  me sometime while I review it. I have a cold 🤧 so might be slow to respond for a little while.

**Comment by dima-bzz** [comment]
Good. Get well. If you write anything, we will discuss the improvements.

**Comment by dima-bzz** [comment]
1. I made an automatic calculation of the panel width depending on authorization. 
2. Added the saving of the hidden panel in localstorage. 
3. Moved the logout button to the panel and changed its display when hiding the panel. 
4. Removed the margins for logout button in the mobile version. (This is temporary until I take out the buttons in the menu)

**Comment by amir20** [comment]
I am feeling slightly better so giving some feedback. 

I deleted `localstorage` and this is what I get as the default view:

Screen Shot 2022-11-08 at 9 20 47 AM

Still feels tight and not proper spacing.  Also I use a Mac by default but by default on Windows this will cause scroll bars. Which is something to be **avoided** for default view. 

I think it's time to think of doing something _different_. It doesn't all have to be in one line. I did a quick hack moved everything to a new line like so: 
Screen Shot 2022-11-08 at 9 28 06 AM

Also made the icons a little smaller.

Getting this UX right will have big value for people first time using Dozzle. And data shows that I am getting about 200 install per 30 minutes. So I am being very careful of the spacing and visual design. 

Screen Shot 2022-11-08 at 9 31 06 AM


Let me know what you think?

File: assets/layouts/default.vue [hunk 1]
```diff
@@ -74,6 +85,9 @@ function showFuzzySearch() {
     active: true,
   });
 }
+function collapse() {
```
Comment (amir20) [inline assets/layouts/default.vue:88]: I am not seeing why this moved to it's own function.

File: assets/composables/settings.ts [hunk 2]
```diff
@@ -69,7 +72,13 @@ const softWrap = computed({
   set: (value) => (settings.value.softWrap = value),
 });
 
+const collapseNav = computed({
```
Comment (amir20) [inline assets/composables/settings.ts:75]: Ah I see, did you make this a setting? That's cool. Send that in a different PR.

File: assets/layouts/default.vue [hunk 3]
```diff
@@ -112,11 +126,26 @@ function onResized(e: any) {
     left: -40px;
     width: 60px;
     padding-left: 40px;
+    color: white;
```
Comment (amir20) [inline assets/layouts/default.vue:129]: Something doesn't sound right here. Colors shouldn't be hardcoded. Dozzle has light and dark mode. Can you make sure this is the right thing? Doing collapse in light mode looks like this

Screen Shot 2022-11-08 at 9 37 53 AM

That looks broken.

**Review by amir20 — CHANGES_REQUESTED** [review]

File: assets/layouts/default.vue (hunk 3 above)
Comment (amir20) [reply assets/layouts/default.vue:129]: Looks like in master I have bug. The background also turns black. Let's see if we can fix that in this PR. 

Is your intention to have the signout stick out? Why? Isn't the purpose of collapse to hide all buttons. User can uncollapse and click it if needed. No reason to take space.

Comment (dima-bzz) [reply assets/layouts/default.vue:129]: If there is no need to show the logout button when the panel is collapsed, then you can remove it and leave it only on the panel.

Comment (dima-bzz) [reply assets/layouts/default.vue:129]: I will try to fix the error with the button color.

File: assets/layouts/default.vue (hunk 1 above)
Comment (dima-bzz) [reply assets/layouts/default.vue:88]: There was a compilation error, so I took it out to the function.

File: assets/layouts/default.vue (hunk 3 above)
Comment (amir20) [reply assets/layouts/default.vue:129]: thank you ❤️

**Comment by dima-bzz** [comment]
@amir20 Can you tell me how you reduced the buttons ?

**Comment by amir20** [comment]
@dima-bzz Try adding `is-small` to `.button` class.

**Comment by dima-bzz** [comment]
@amir20 What do you say?

**Comment by amir20** [comment]
I did a quick pull. It looks better. What do you think? Do you like it more?

File: assets/layouts/default.vue [hunk 4]
```diff
@@ -112,7 +114,8 @@ function onResized(e: any) {
     left: -40px;
     width: 60px;
     padding-left: 40px;
-    background: rgba(0, 0, 0, 0.95);
+    color: var(--text-strong-color);
```
Comment (amir20) [inline assets/layouts/default.vue:117]: ❤️

File: assets/pages/index.vue [hunk 5]
```diff
@@ -1,13 +1,9 @@
...
   <div>
-    <section class="hero is-small mt-4">
-      <div class="hero-body">
-        <div class="container">
-          <div class="columns">
-            <div class="column is-narrow" v-if="secured">
-              <a class="button is-primary is-small" :href="`${base}/logout`">{{ $t("button.logout") }}</a>
-            </div>
-          </div>
+    <section class="is-small mt-1" v-if="secured && isMobile">
```
Comment (amir20) [inline assets/pages/index.vue:3]: You said you will come back to this, right? We can do something similar in mobile and have a row for all the buttons.

**Review by amir20 — APPROVED** [review]

**Comment by amir20** [comment]
Files look good. I can merge if you are ok with it.

File: assets/pages/index.vue (hunk 5 above)
Comment (dima-bzz) [reply assets/pages/index.vue:3]: You can leave the button showing only in the mobile version. But I already have a revision on the buttons for the mobile menu.

**Comment by dima-bzz** [comment]
I don't mind

**Comment by amir20** [comment]
Merging and fixing int tests locally.

**Comment by amir20** [comment]
Something was broken when I merged. Had to fix it with d6e87a7

**Comment by amir20** [comment]
I should have not ignored the int tests. :) https://github.com/amir20/dozzle/actions/runs/3422645762/jobs/5700295282
